src/sonic-config-engine/minigraph.py

```diff
@@ -615,34 +726,59 @@ def parse_xml(filename, platform=None, port_config_file=None):
         if child.tag == str(docker_routing_config_mode_qn):
             docker_routing_config_mode = child.text
 
-    (ports, alias_map) = get_port_config(hwsku, platform, port_config_file)
+    (ports, alias_map, alias_asic_map) = get_port_config(hwsku=hwsku, platform=platform, port_config_file=port_config_file, asic=asic_id)
     port_alias_map.update(alias_map)
+    port_alias_asic_map.update(alias_asic_map)
+
     for child in root:
```

Comment:
suggest to reorg this portion of code for better clarity

if asic_name is None:
    check dpg/cpg/
else:
    check dpg/cpg/...

Reply:
Fixed as per suggestion